Makefile.am

```diff
@@ -63,7 +63,6 @@ bin_PROGRAMS = \
     tools/tpm2_getrandom \
     tools/tpm2_hash \
     tools/tpm2_hmac \
-    tools/tpm2_import \
```

Comment:
This is fine, just nuke this tool.

Reply:
@williamcroberts Ok, I'll remove the this tool then. I guess that's OK since the implementation is tightly coupled with the SAPI 2.0 version.